hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieWriteStat.java

```diff
@@ -68,6 +69,30 @@
    */
   private long totalWriteErrors;
 
+  /**
```

Comment:
Considered making this Optional<> but that complicates converting to the corresponding AVRO model since the AVRO auto-generated doesn't have optional.

Reply:
if there is @Nullable annotation, you could add that and that would be swell for code readability

Reply:
done.